inc/containerdisplaycondition.class.php

```diff
@@ -362,7 +362,7 @@ public static function removeBlackListedOption($array, $itemtype_class)
     }
 
 
-    public function computeDisplayContainer($item, $container_id)
+    public function computeDisplayContainer($item, $container_id, $params = null)
```

Comment:
1. `$input` seems to be a more appropriate name.
2. It should always be passed as an array.
```suggestion
    public function computeDisplayContainer($item, $container_id, array $input)
```